type/src/ai-card.ts

```diff
@@ -0,0 +1,197 @@
+// --8<-- [start:AICard]
+/**
+ * The unified AI Card object for a single agent.
+ */
+export interface AICard {
+  /**
+   * Declares the schema this card adheres to.
+   * e.g., "https://ai-agent-protocol.org/ai-card/v1/schema.json"
+   */
+  $schema: string;
+
+  /**
+   * The version of the AI Card specification itself.
+   */
+  specVersion: string;
+  
+  /**
+   * The primary verifiable ID for the agent (e.g., DID).
+   */
+  id: string; 
+  
+  /**
+   * A human-readable name for the agent.
+   */
+  name: string;
+
+  /**
+   * A short, human-readable description of the agent's purpose.
+   */
+  description: string;
+
+  /**
+   * A direct URL to the agent's logo image.
+   */
+  logoUrl?: string;
+
+  /**
+   * A list of keywords to aid in discovery.
+   */
+  tags?: string[];
+
+  /**
+   * Information about the entity (company or individual) that published this agent.
+   */
+  publisher: Publisher;
+
+  /**
+   * Holds all trust, compliance, and identity information.
+   */
+  trust: Trust;
+  
+  /**
+   * An array of all interaction protocols this agent supports.
+   * This is the core of the extensibility model.
+   */
+  services: BaseService[];
+
+  /**
+   * An ISO 8601 timestamp of when the agent was first published.
+   */
+  createdAt: string; 
+
+  /**
+   * An ISO 8601 timestamp of when this card was last updated.
+   */
+  updatedAt: string; 
+
+  /**
+   * An open "black box" for any other non-standard metadata.
+   */
+  metadata?: Record<string, any>;
+}
+// --8<-- [end:AICard]
+
+// --- CORE COMPONENTS ---
+
+// --8<-- [start:Publisher]
+/**
+ * Defines the entity (company, individual) that published the agent.
+ */
+export interface Publisher {
+  /**
+   * A verifiable ID for the publisher, e.g., a DID or organization ID.
+   */
+  id: string;
+
+  /**
+   * The human-readable name of the publisher.
+   */
+  name: string;
+
+  /**
+   * A URL to a verifiable credential (e.g., JWT) proving the publisher's identity.
+   */
+  attestation?: string;
```

Comment:
What is the reason for using just a string here, but using the Attestation type under Trust?  Could the Attestation type be reused here?